tools/nl/embeddings/input/bio/sheets_svs.csv

```diff
@@ -37,10 +33,7 @@ geneID,geneID;gene id
 ncbiProteinAccessionNumber,ncbiProteinAccessionNumber;NCBI protein accession number
 alleleOrigin,alleleOrigin;Variant allele origin;Origin of variant allele
 alleleType,alleleType;The allele of a genetic variant observed within a population;Type of allele
-ncbiDNASequenceName,ncbiDNASequenceName;NCBI defined segment of DNA sequence name;Name used by NIH NCBI to refer to a segment of DNA sequence
-imageUrl,imageUrl;url to an image of what the biological specimen looks like;what the entity looks like
-genomicCoordinates,genomicCoordinates;genomic coordinates
-availableStrength,availableStrength;dose approved for a drug
+hasGenomicCoordinates,genomicCoordinates;genomic coordinates

```

Comment:
it shouldn't be necessary to have both genomicCoordinates and genomic coordinates as sentences

Reply:
removed genomicCoordinates, thanks!